AdblockPlusSafariExtension/AdblockPlus+Parsing.m

```diff
@@ -26,8 +26,16 @@
 #include <stdlib.h>
 #include <string.h>
 
+typedef NS_ENUM(NSUInteger, AdblockPlusFilterListsType) {
+  AdblockPlusFilterListsTypeVersion1,
+  AdblockPlusFilterListsTypeVersion2
+};
+
 typedef struct
 {
+  BOOL writingEnabled;
```

Comment:
Writing is always disable, when parser is processing incompatible parts of the JSON tree.